internal/batches/executor/changeset_specs_test.go

```diff
@@ -0,0 +1,313 @@
+package executor
```

Comment:
Nothing changed in this file. I only moved the test here from `executor_test.go`